app/main.py

```diff
@@ -67,25 +55,44 @@ def empty_directory():
             traceback.print_exc()
 
 
-@app.route('/upload', methods=['POST'])
-def upload():
+def output(response_type, message, download_link):
+    if download_link == '':
+        response = [
+            {
+                'type': response_type,
+                'message': message
+            }
+        ]
+    else:
+        response = [
+            {
+                'type': response_type,
+                'message': message,
+                'download_link': download_link
```

Comment:
Why is download link being provided instead of the pdf file as response??

Reply:
The link is provided just as the rest api specifications and common practice. It is what the the web app delivers as well. We can't explicitly send the file as part of an api :P

Reply:
The download link is the link to download the pdf file.

Reply:
Ok @abishekvashok

Reply:
Sure @S2606